sys/include/net/gnrc/ipv6/netif.h

```diff
@@ -331,14 +331,14 @@ typedef struct {
      *          gnrc_ipv6_netif_t::reach_time_base microseconds devided by 10.
      *          Can't be greater than 1 hour.
      */
-    timex_t reach_time;
+    uint32_t reach_time;
 
     /**
      * @brief   Time between retransmissions of neighbor solicitations to a
      *          neighbor.
      *          The default value is @ref GNRC_NDP_RETRANS_TIMER.
      */
-    timex_t retrans_timer;
+    uint32_t retrans_timer;
```

Comment:
Both `reach_time` and `retrans_timer` need to be of `uint64_t` or handled as milliseconds altogether. Both values are [32-bit millisecond values](https://tools.ietf.org/html/rfc4861#page-21) in the RFC, but used as microseconds here.